app/models/public_body/calculated_home_page.rb

```diff
@@ -0,0 +1,52 @@
+# Guess the home page based on the request email domain.
+module PublicBody::CalculatedHomePage
+  extend ActiveSupport::Concern
+
+  included do
+    cattr_accessor :excluded_calculated_home_page_domains, default: %w[
+      aol.com
+      btinternet.com
```

Comment:
```suggestion
      aol.com
      aol.co.uk
      btinternet.com
      btconnect.com
      btopenworld.com
      talk21.com
      sky.com
      ntlworld.com
      virginmedia.com
      blueyonder.co.uk
      tiscali.co.uk
      talktalk.net
```

Cumulatively, we have a sizeable number across each[^1], so, in our own way - we gotta catch them all.

[^1]: Particularly the four BT ones, but the other consumer ISPs also add a fair amount of bulk… they are a favourite for Town and Parish councils!